fixtures/litecraft/beamx-7.json

```
@@ -0,0 +1,172 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Beamx.7",
+  "shortName": "Bx.7",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Pyronaut"],
+    "createDate": "2024-07-12",
+    "lastModifyDate": "2024-07-12"
+  },
+  "comment": "Par Can",
+  "links": {
+    "manual": [
+      "https://www.huss-licht-ton.de/images/products_download/User_Manual_21947_1.pdf"
+    ],
+    "productPage": [
+      "https://lmp.de/produkteshop/scheinwerfer-und-komponenten/4961/litecraft-beamx.7-ip"
+    ]
+  },
+  "physical": {
+    "dimensions": [216, 150, 231],
+    "weight": 2.3,
+    "power": 46,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "Multicolour LED"
+    },
+    "lens": {
+      "degreesMinMax": [10, 10]
+    }
+  },
+  "availableChannels": {
+    "Color Macros": {
+      "capability": {
+        "type": "ColorPreset"
```

```suggestion
        "type": "ColorPreset",
        "helpWanted": "What colors, at what values?"
```

The manual simply says "43 pre-programmed colors combinations available".